apps/www/lib/images/AssetImage.tsx

```diff
@@ -1,12 +1,32 @@
+import { useMemo } from 'react'
 import Image, { ImageProps } from 'next/image'
 import assetServerImageLoader from './assetServerImageLoader'
+import { isDev } from '../constants'
 
 /**
  * AssetImage is a wrapper around the Next.js Image component
  * that uses the asssets-server image-loader.
  */
-const AssetImage = (props: Omit<ImageProps, 'loader'>) => (
-  <Image {...props} loader={assetServerImageLoader} />
-)
+const AssetImage = (props: Omit<ImageProps, 'loader'>) => {
+  // Local static images that are not yet on the asset-server
+  // can be loaded directly from the local file system while in development.
+  const loadStaticFromLocal = useMemo(() => {
+    if (
+      isDev &&
+      typeof props?.src === 'string' &&
+      props.src.startsWith('/static')
```

Comment:
Nit: `props` is not optional, so no need for `?`?

Not sure if we should check for the other possible types (`StaticRequire | StaticImageData`) here too. I.e. `src.default.src` or `src.src` (oh my).

Reply:
Also, can we just check for `/` instead of `/static`? If the image's at `public/foo.jpg` that would be the case (`/static` is not necessary anymore, no?)

Reply:
Will support StaticImageData but not StaticImageRequire as it is not exported from next anyways